### ISSUE TITLE
Project board performance is too slow/times out/or crashes

### ISSUE THREAD

**Issue by SAUMILDHANKAR** [hackforla/website#3031]
### Overview
As team leads, we would like our project board to be responsive. In order to ask GitHub for help, we need to identify all the types of freezing and crashing so that we can communicate it effectively to GitHub.

### Detail
Anyone who experiences freezing or crashing, should detail their experience, how to replicate and an animated GIF if possible. See examples below for how to document

- Bonnie
  - https://github.com/hackforla/website/issues/3031#issuecomment-1090720745
  - https://github.com/hackforla/website/issues/3031#issuecomment-1090726358
  - https://github.com/hackforla/website/issues/3031#issuecomment-1090728357

### Action Items
- [x] Identify when our [project board's](https://github.com/hackforla/website/projects/7) is freezing, crashing, times out, or is too slow:
    - [Moving issues from one column to another](https://github.com/hackforla/website/issues/3031#issuecomment-1090720745)
    - [Waiting on GitHub message](https://github.com/hackforla/website/issues/3031#issuecomment-1090726358)
    - [Filtering issues by clicking on a label and then trying to scroll](https://github.com/hackforla/website/issues/3031#issuecomment-1099766894)
    - Opening project board in more than one tab
    - Searching for a keyword 
- [x] Document our project board's key metrics such as number of cards, number of issues, columns, team members, number of automations (and one line description), etc. that might affect the performance in a comment below.
- [x] In a comment below, describe the problem you are encountering with the Project Board (trying to be as factual as possible with a few words)
- [x] Reach out to the GitHub customer service or open a [new ticket](https://support.github.com/tickets/personal/0) or [submit a bug report](https://support.github.com/contact/bug-report) for recommendations, remedies and other options
- [x] Discuss their response our all team meeting 

### Resources/Instructions
- [Link to our project board](https://github.com/hackforla/website/projects/7)
- [Link to the project board, we created to move issues to](https://github.com/hackforla/website/projects/14)
- [Screen recording gif making tool (all in one)](https://www.cockos.com/licecap/)
- [GitHub support](https://support.github.com/contact?tags=rr-general-technical)
- [GitHub: open a new ticket](https://support.github.com/tickets/personal/0)
- [GitHub: submit a bug report](https://support.github.com/contact/bug-report)

**Comment by ExperimentsInHonesty** [comment]
Image of trying to move an issue from one column to another with a timer
![screenrecordtest7](https://user-images.githubusercontent.com/37763229/162060917-2b9e405b-5480-41ef-acda-f665f199e7a1.gif)

**Comment by ExperimentsInHonesty** [comment]
Image showing that the board goes into a cycle of loading when you use the scroll bar (starts 16 seconds in, and continues to the 1 min mark)
![screenrecordtest8](https://user-images.githubusercontent.com/37763229/162061991-f7d7d02d-9961-43df-aada-20859e246d6f.gif)

**Comment by ExperimentsInHonesty** [comment]
The speed test on the computer  for the above screenshots
![image](https://user-images.githubusercontent.com/37763229/162062603-8626c12e-3b04-44c4-a14c-1d52f8992f9c.png)

**Comment by JessicaLucindaCheng** [comment]
- Jessica's Availability: 3 hours
- ETA to contact GitHub: Sunday, April 17, 2022

**Comment by SAUMILDHANKAR** [comment]
Availability: 5 hours
ETA to contact GitHub: Sunday, April 17, 2022

**Comment by SAUMILDHANKAR** [comment]
Video showing the board becomes unresponsive if you try to scroll after filtering the board with an existing label
- Internet speed test: Download 48 Mbps, Upload 11 Mbps

https://user-images.githubusercontent.com/843538/163502531-70107070-2614-4fec-920f-29baa3036ac1.mp4

**Comment by SAUMILDHANKAR** [comment]
Hackf for LA website team's project board key metrics:
- Number of columns: 15
- Number of cards: 1235
- Number of contributors: 177
- Number of workflows/automations/GitHub actions: 15
- Number of labels: 116

**Comment by SAUMILDHANKAR** [comment]
### Bug report draft
- From: Saumil Dhankar
- Organization: Hack for LA
- Subject: Project board performance is too slow/times out/or crashes
- What type of issue is this?: Permanent - Occurring repeatedly
- Is this issue blocking you?: Blocking
- Last known working date - Around 6 months ago.
- What issue are you facing?
Following project boards both of which are part of Hack for LA organization are facing the below mentioned issues: [Project Board of Hack for LA's website team](https://github.com/hackforla/website/projects/7), [Project Board of Hack for LA's CTI website frontend team](https://github.com/civictechindex/CTI-website-frontend/projects/1)
    - Project board crashes/slows down/times out while performing tasks such as [moving issues from one column to another](https://github.com/hackforla/website/issues/3031#issuecomment-1090720745), [waiting on GitHub message](https://github.com/hackforla/website/issues/3031#issuecomment-1090726358), [filtering issues by clicking on a label and then trying to scroll](https://github.com/hackforla/website/issues/3031#issuecomment-1099766894) 
    - Project page stays unresponsive and takes a long time to return to normalcy.
    - Some of the work such as adding label or moving an issue to a different column is lost as well due to these crashes.

  We have tried moving a lot of older issues off the project board to a new project board, but that didn't improve the performance either.
  [Key project metrics](https://github.com/hackforla/website/issues/3031#issuecomment-1100605057) for [Project Board of Hack for LA's website team](https://github.com/hackforla/website/projects/7). Please let us know if you recommend keeping the number of issues, automations etc. on the [Project Board of Hack for LA's website team](https://github.com/hackforla/website/projects/7) down to a certain number to experience improved performance.
- What are the steps to reproduce this?
    - [Moving issues from one column to another](https://github.com/hackforla/website/issues/3031#issuecomment-1090720745)
    - [Waiting on GitHub message](https://github.com/hackforla/website/issues/3031#issuecomment-1090726358)
    - [Filtering issues by clicking on a label and then trying to scroll](https://github.com/hackforla/website/issues/3031#issuecomment-1099766894)
    - When the project board is open in multiple tabs
    - Using the search functionality to narrow the search on the Project Board by assignee, label etc.
- Can you provide any logs? i.e. response headers, output
    - Links to the project boards: [Project Board of Hack for LA's website team](https://github.com/hackforla/website/projects/7), [Project Board of Hack for LA's CTI website frontend team](https://github.com/civictechindex/CTI-website-frontend/projects/1)
    - Links to screenshots and video: [Moving issues from one column to another](https://github.com/hackforla/website/issues/3031#issuecomment-1090720745), [Waiting on GitHub message](https://github.com/hackforla/website/issues/3031#issuecomment-1090726358), [Filtering issues by clicking on a label and then trying to scroll](https://github.com/hackforla/website/issues/3031#issuecomment-1099766894)

**Comment by JessicaLucindaCheng** [comment]
Unassigned myself because I didn't do any work on this issue since I was sick for a few days. Thus, Saumil has been doing all the work and looks is almost done with it.

**Comment by SAUMILDHANKAR** [comment]
Waiting for response from GitHub support on Ticket  # 1594518

**Comment by SAUMILDHANKAR** [comment]
Update: Ticket has been updated and the issue has been escalated to the GitHub engineering team. Forwarded the latest update on webadmin@hackforla.org.
Blocker: Waiting to hear back from the engineering team
ETA: 1 week.
Availability: 2 hours this week.

**Comment by ExperimentsInHonesty** [comment]
@SAUMILDHANKAR please paste the emails here as well so we can easily follow along.

**Comment by SAUMILDHANKAR** [comment]
@ExperimentsInHonesty GitHub support team has suggested to try out the new style project board which is in beta version. For the website team, it might not be the best step as we have a lot of issues and it might be very inconvenient if it doesn't go well. Should be tried out on a smaller project first. Look forward to your feedback on this. Below is the entire conversation I have had with the GitHub support team so far. :

**Guy (GitHub Support)
 May 10, 2022, 1:43 PM UTC**

Hi Saumil,

Thanks for your patience whilst we've been looking into this.

I'm afraid I don't have specific target numbers for the maximum number of issues to have on a board, as there are multiple factors that impact performance.

Our engineering team is currently working on a migration tool to convert original-style project boards into [new style projects (beta) boards](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/about-projects), which should be more performant. Would you be interested in being a test user for this new tool and convert your projects to the new style?

Cheers,

Guy

🐸 Find answers to common questions and learn with other GitHub users in our new [GitHub Community Forum](https://github.community/) 🐸

	
**Saumil Dhankar
 May 2, 2022, 9:26 PM UTC**

Hello team,

I am re-opening this since we haven't found a resolution to the issue or any closing remarks. However, would like to share an update on some of the things that we have been trying on our end.

We reduced the total number of issues on one of our project boards: [Project Board of Hack for LA's CTI website frontend team](https://github.com/civictechindex/CTI-website-frontend/projects/1) from 709 to 158 and the performance has improved. So, just want to check if the engineering team could recommend a cut-off for the total number of issues to be on the board (or any other useful metric), that would be very helpful and we can try implementing that on our other project boards as well.
We are also planning to reduce the total number of issues on all our project boards in general, but again would be very helpful to know if there is a specific number we can target.
Please let us know if there are any questions or if any point needs more clarity. Would really appreciate your comments or help on this issue. Thank you.

	
**Elle (GitHub Support)
 Apr 27, 2022, 4:29 AM UTC**

Hi Saumil Dhankar,

Thanks for the detailed report and I'm sorry you have experienced this problem.

I have escalated your report with our engineering team for further investigation.

If you think of anything else you would like to add to the report, please do let me know.

I can't make any promises around specific changes or updates, but please trust that your report is in good hands.

We recommend that you keep an eye on our [Changelog](https://github.blog/changelog/) and our [GitHub Blog](http://github.blog/) where we publish updates on everything we ship.

If you need anything else at all, please feel free to reach out again.

All the best,
Elle

	
**Saumil Dhankar
 Apr 20, 2022, 10:22 PM UTC**

Description

Following project boards both of which are part of Hack for LA organization are facing the below mentioned issues: [Project Board of Hack for LA's website team](https://github.com/hackforla/website/projects/7), [Project Board of Hack for LA's CTI website frontend team](https://github.com/civictechindex/CTI-website-frontend/projects/1)

Project board crashes/slows down/times out while performing tasks such as [moving issues from one column to another](https://github.com/hackforla/website/issues/3031#issuecomment-1090720745), [waiting on GitHub message](https://github.com/hackforla/website/issues/3031#issuecomment-1090726358), [filtering issues by clicking on a label and then trying to scroll](https://github.com/hackforla/website/issues/3031#issuecomment-1099766894)
Project page stays unresponsive and takes a long time to return to normalcy.
Some of the work such as adding a label or moving an issue to a different column is lost as well due to these crashes.
We have tried moving a lot of older issues off the project board to a new project board, but that didn't improve the performance either.
[Key project metrics](https://github.com/hackforla/website/issues/3031#issuecomment-1100605057) for [Project Board of Hack for LA's website team](https://github.com/hackforla/website/projects/7). Please let us know if you recommend keeping the number of issues, automations etc. on the [Project Board of Hack for LA's website team](https://github.com/hackforla/website/projects/7) down to a certain number to experience improved performance.

Reproduction Steps

The issue is experienced in each of the following cases:

[Moving issues from one column to another](https://github.com/hackforla/website/issues/3031#issuecomment-1090720745)
[Waiting on GitHub message](https://github.com/hackforla/website/issues/3031#issuecomment-1090726358)
[Filtering issues by clicking on a label and then trying to scroll](https://github.com/hackforla/website/issues/3031#issuecomment-1099766894)
When the project board is open in multiple tabs
Using the search functionality to narrow the search on the Project Board by assignee, label etc.
Fault Type

Permanent - Occurring repeatably

Blocking?

Blocking

Last working date

01 September

Logs

Links to the project boards: [Project Board of Hack for LA's website team](https://github.com/hackforla/website/projects/7), [Project Board of Hack for LA's CTI website frontend team](https://github.com/civictechindex/CTI-website-frontend/projects/1)
Links to screenshots and video: [Moving issues from one column to another](https://github.com/hackforla/website/issues/3031#issuecomment-1090720745), [Waiting on GitHub message](https://github.com/hackforla/website/issues/3031#issuecomment-1090726358), [Filtering issues by clicking on a label and then trying to scroll](https://github.com/hackforla/website/issues/3031#issuecomment-1099766894)

**Comment by ExperimentsInHonesty** [comment]
Please pass this message along

Hi GitHub.

First let me say - I love  you!  Seriously!  I think I am your biggest fan.

I like the idea of the beta project boards.  But I don't think that you understand just how powerful your issue labels are.  Because if you did, you would not have made a tool that completely ignores what they can do and removed the ability to filter the board by them.

We will be interested in using the new projects boards when that feature is integrated.

Here are some examples of the way that we use the label now.

When interdisciplinary teams work together, it's best for Product management for all the issues to be on one project board.  Teams sometimes want to each make their own project board.  But we can tell them, that all they have to do, is sort an issue by the role label we have created.  See these examples:
- Front End view of the board: https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22
- User Research view of the board: https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+user+research%22

Another way we use them, is to make sure we have enough issues for our developer success ladder (where we give them increasingly harder issues until they fully understand our code base.
We check weekly to see if we have enough issues with the  "good first issue" label, so that we can accommodate all the new developers joining our team: https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22#column-7198257.   But we also check our other size labels (soon to be called complexity instead of size) to ensure that we decompose larger issues when needed.
Size: Good second issue
Size: Small
Size: Medium https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22size%3A+medium%22#column-7198257
Size: Large

I have more examples, but I hope this will help you understand why we can't adopt the new layout until the filter feature is added.

Best,

Bonnie Wolfe, Executive Director, Hack for LA
GitHub handle: Experimentsinhonesty
Personal email: experimentsinhonesty@gmail.com
Organizational email: bonnie@hackforla.org

**Comment by ExperimentsInHonesty** [comment]
Please pass this message along

Hi GitHub.

First let me say - I love  you!  Seriously!  I think I am your biggest fan.

I like the idea of the beta project boards.  But I don't think that you understand just how powerful your issue labels are.  Because if you did, you would not have made a tool that completely ignores what they can do and removed the ability to filter the board by them.

We will be interested in using the new projects boards when that feature is integrated.

Here are some examples of the way that we use the label now.

Another way we use them, is to make sure we have enough issues for our developer success ladder (where we give them increasingly harder issues until they fully understand our code base.
We check weekly to see if we have enough issues with the  "good first issue" label, so that we can accommodate all the new developers joining our team: https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22#column-7198257.   But we also check our other size labels (soon to be called complexity instead of size) to ensure that we decompose larger issues when needed.
Size: Good second issue
Size: Small
Size: Medium https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22size%3A+medium%22#column-7198257
Size: Large

I have more examples, but I hope this will help you understand why we can't adopt the new layout until the filter feature is added.

Best,

Bonnie Wolfe, Executive Director, Hack for LA
GitHub handle: Experimentsinhonesty
Personal email: experimentsinhonesty@gmail.com
Organizational email: bonnie@hackforla.org

**Comment by ExperimentsInHonesty** [comment]
We are interested, but right now the beta boards are only at the org level.  If we were part of the conversion scrip beta would it be available at the repo level?

When the script is used, does it move the issue from the current board to the new version or does it just add the project to the issue?  We would want to have it on both the old and the new, so that we could compare our experience.

As them what the other factors are.

**Comment by SAUMILDHANKAR** [comment]
Progress: Reached out to GitHub with the above suggested questions. Got a response that the engineering team is looking into it and will get back.
Blockers: No blockers
Availability: 1 hr
ETA: Depending on response from the support team. There should be an update by next week.

**Comment by SAUMILDHANKAR** [comment]
@ExperimentsInHonesty @JessicaLucindaCheng Below are the emails since our last discussion on May 19th. Engineering team has proposed to set up a call for help on migration. 
	
Guy (GitHub Support)

May 31, 2022, 10:52 AM UTC

Hi Saumil,

Thanks for your patience while the engineering team have worked on this.

They have a few answers for you, and would also like to propose having a call with you.

You mentioned that there are multiple factors that can impact the performance. Can you share with us these factors so we can consider planning accordingly?
The version of Projects you are using is under maintenance mode and we don't currently have the resources to dig into what's impacting performance. We recommend that Hack of LA scale their project board down (which, it looks like you're doing) or migrate to Projects beta. We'd be happy to set up a call to help you migrate.

Right now the beta boards are only at the org level. If we were part of the conversion script beta, would it be available at the repo level as well?
You would have to create the project at the org level, associate the project to the repo and then you can access the project at the repo level.

When the script is used, does it move the issue from the current board to the new version or does it just add the project to the issue? We would want to have it on both the old and the new, so that we could compare our experience.
You would have access to the old and new board so that you can compare the experience.

Let me know if they would like me to set up a call with them!

Cheers,

Guy

🦕 Find answers to common questions and learn with other GitHub users in our new [GitHub Community Forum](https://github.community/) 🦕

==================================
	
Saumil Dhankar

May 28, 2022, 8:03 PM UTC

Hi Guy, just wanted to follow up if there is any update from the engineering team. Please let me know.

Thanks, Saumil

==================================
	
Guy (GitHub Support)

May 16, 2022, 11:17 AM UTC

Hi Saumil,

I'll pass your questions on to the engineering team and get back to you when we have a response!

Cheers,

Guy

🦧 Find answers to common questions and learn with other GitHub users in our new [GitHub Community Forum](https://github.community/) 🦧

=================================
	
Saumil Dhankar

May 15, 2022, 1:03 AM UTC

Hi Guy,

Thank you for your update. We are interested in being the test user for the migration tool but based on the discussion I had with our organizational leadership; it would be really helpful if you could provide us some guidance on the following points:

You mentioned that there are multiple factors that can impact the performance. Can you share with us these factors so we can consider planning accordingly?
Right now the beta boards are only at the org level. If we were part of the conversion script beta, would it be available at the repo level as well?
When the script is used, does it move the issue from the current board to the new version or does it just add the project to the issue? We would want to have it on both the old and the new, so that we could compare our experience.
Please let us know your comments and the next steps. Thanks again for all your help.

**Comment by SAUMILDHANKAR** [comment]
Progress: Jessica was able to create the project at the org level, associate the project to the repo and then access the project at the repo level. Jessica and I will demo it to Bonnie on Sunday (or later as time permits) and based on her feedback will decide on the next steps.
Blockers: No blockers
Availability: 1 hr
ETA: Depends on the discussion on Sunday.

**Comment by JessicaLucindaCheng** [comment]
Notes on how to create the project at the org level, associate the project to the repo and then access the project at the repo level

### Action Items
Figure out the answers to these questions
- [x] How to create the project at the org level?
  - Ans:
    - Go to https://github.com/hackforla
    - Click on "Projects" tab
    - Click on green "Create project" button on the right of your screen
- [x] How to get a project at the org level associated with a repo, such as the Website repo?
  - Ans:
    - After creating a new beta project board, go to the repo (such as the Website repo) you want the board associated with
    - Make sure the "Projects (Beta)" is selected on the left side
    - Scroll down and click on the "Add Project" button. A dropdown with a search field should show up.
    - In the search field, type in the name of the beta board in the search field and results matching your search should show up. Click on the beta project you want associated with the repo.
- [x] How to access the project at the repo level?
  - Ans:
    - After associating the project beta with a repo, you can access the project beta by going to the repo (such as the Website repo).
    - Make sure the "Projects (Beta)" is selected on the left side.
    - Then scroll down and you should see the project beta board listed.
    - Note: Who can see the board depends on what you set in the "Manage access" area of the project beta, such as public or private project, base role access, and who and what role people/teams have. (To get to "Manage access", go the to the project beta board, click on the 3 dots (...) on the top right of the board --> Settings --> Manage Access)

**Comment by SAUMILDHANKAR** [comment]
Did a demo today with Bonnie on how to create the project at the org level, associate the project to the repo and then access the project at the repo level. Looks good for the next steps.

Decided to go ahead and set up a call with the GitHub support team to discuss the next steps for migration. Would bring up in the Dev/PM meeting tomorrow and finalize the meeting times with Tamara, Jessica, Saumil and Bonnie. (Bonnie may not join the meeting tomorrow depending on availability but go ahead anyways). Invite the merge team members for the call with the GitHub support team as well.

**Comment by SAUMILDHANKAR** [comment]
Reached out to GitHub to set up a meeting on migration from classic to beta board. Below is what I wrote yesterday:

> Hi Guy,
> 
> Thanks a lot for sharing all the information. Our team is interested in scheduling up a call with the GitHub engineering team for help on migrating to Project beta. 
> 
> Please let me know the next steps regarding setting up a call with the GitHub engineering team. We can set up a zoom meeting for the session as well, if required.
> 
> Below are the times which would work for our team:
> 
>
> 10 AM Pacific time on Tuesday, June 14, 2022
> 1:30 PM Pacific time on Wednesday, June 15, 2022
> 2 PM Pacific time on Thursday, June 16, 2022
> 2 PM Pacific time on Monday, June 20, 2022
> 
> Please let me know if any of the above times would work for the GitHub engineering team or if they would like to suggest some other time for the call.
>
> 
> Following members of our organization are interested in joining the call:
> 
> Bonnie Wolfe (Email: [bonnie@hackforla.org](mailto:bonnie@hackforla.org))
> Jessica Lucinda Cheng (Email: [Cheng.JessicaLucinda@gmail.com](mailto:Cheng.JessicaLucinda@gmail.com))
> Saumil Dhankar (Email: [babyyodanemo2020@gmail.com](mailto:babyyodanemo2020@gmail.com))
> Tamara Snyder (Email: [tamara.f.snyder@gmail.com](mailto:tamara.f.snyder@gmail.com))
> FYI: As per your last message, our team was able to create the project at the org level, associate the project to the repo and then access the project at the repo level. We also had discussions (within our HackforLA organization) on the next steps and hence the delay in getting back.
> 
> Look forward to hearing from you and thank you for all your help.
> 
> Regards,
> Saumil

**Comment by JessicaLucindaCheng** [comment]
Edit: Notes from here are copied to #3265, so that all notes related to moving to Projects beta are in one issue. 

Notes from June 15 meeting with Josie from GitHub
- The beta projects have an issue limit of 1200. Currently, we have 1195. Thus, the first thing we need to do is Archive some issues or somehow reduce the number.
-  On our current project board, Bonnie discusses how we have cards with filters (by role and complexity) and as info/descriptions at the top of each column. Since cards become draft issues, we need to figure out how we going to have
  - filters (by role and complexity)
  - the info/description cards (though Bonnie said maybe we could move some of these into the Wiki)
- You can archive one issue at a time (or a whole column)
  - [ ] May need to show Bonnie how to archive one issue at a time
- Want to figure out how to evaluate if another person (of a certain role) needs to be recruited to the team by filtering by role and looking at size/pts, and seeing if there are more than 2 months worth of points.
- Even though currently we have both a classic version and a beta version of the project board, they may not be in sync.
- `Migration feature` to be rolled out soon (maybe in next few days) would enable all members with write-access to be able to migrate the project to beta version.
- If a project is migrated to the beta version second time, the old beta would disappear and get deleted, resulting in loss of all the progress and changes made in the old beta project.
- We were not able to locate column ids or links in the new project beta version and the GitHub team would get back to us on that, but some of our GitHub actions might be impacted because of this. 
- Header of the new project beta has different features than the classic project board and there is separate GitHub team responsible for that. So, it may not be part of the migration release.

**Comment by SAUMILDHANKAR** [comment]
Below is the latest response from the GitHub support team:

> Hi Saumil,
> 
> I understand invitations have been sent out to everyone, so I'll close this ticket and let you communicate directly with the engineering team.
> 
> You can still re-open this ticket again if you need to within 30 days, and if it's more than 30 days you can create a follow up ticket which will be linked to this one, so we don't lose any history.
> 
> Cheers,
> 
> Guy
> 
> 🐸 Find answers to common questions and learn with other GitHub users in our new [GitHub Community Forum](https://github.community/) 🐸

**Comment by JessicaLucindaCheng** [comment]
@SAUMILDHANKAR Bonnie archived a whole bunch of issues in the Done column on the current Project Board and the current Project Board seems to work better. Can this issue be closed then?

Also, I wrote issue #3265 to address migrating from Project classic to beta, since that seems like it will be a bigger/separate issue and might be out of scope for this issue.

**Comment by SAUMILDHANKAR** [comment]
@JessicaLucindaCheng Thanks for writing the issue for migration. I agree follow up items might be out of scope for this issue, so closing this issue. Thanks for all your help.
